lib/configure.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+var debug = require('debug'),
+  log = debug('nemo:log'),
+  confit = require('confit'),
+  _ = require('lodash'),
+  path = require('path'),
+  Promiz = require('./promise'),
+  handlers = require('shortstop-handlers'),
+  yargs = require('yargs'),
+  error = debug('nemo:error');
+
+log.log = console.log.bind(console);
+error.log = console.error.bind(console);
+
+module.exports = function Configure(_basedir, _configOverride) {
+  log('_basedir %s, _configOverride %o', _basedir, _configOverride);
+  var basedir, configOverride;
+  //settle arguments
+  basedir = (arguments.length && typeof arguments[0] === 'string') ? arguments[0] : process.env.nemoBaseDir || undefined;
+  configOverride = (!basedir && arguments.length && typeof arguments[0] === 'object') ? arguments[0] : undefined;
+  configOverride = (!configOverride && arguments.length && arguments[1] && typeof arguments[1] === 'object') ? arguments[1] : configOverride;
+  configOverride = (!configOverride) ? {} : configOverride;
+
+  log('basedir %s, configOverride %o', basedir, configOverride);
+
+  var prom = Promiz();
+
+  //hack because confit doesn't JSON.parse environment variables before merging
```

Comment:
Are you referring to the `JSON.parse` of env variables? If so I had a rather long discussion on including it to confit directly and was pretty satisfied afterwards that it's a bad idea. The only reason it's included is that examples had `grunt-loop-mocha` passing JSON objects as stringified environment variables. Deprecating `grunt-loop-mocha` in examples and focusing on `nemo-runner` (which uses the main configuration files) means this hack could probably just be deprecated and removed by nemo@3.